lib/qtl-data.js

```diff
@@ -0,0 +1,74 @@
+import {
+  TissueScoreHexColor,
+  TissueScoreTailwindColor,
+} from "./tissue-specific-score";
+
+/**
+ * This function get filterred QTL data
+ * @param {Array} data all the QTL data
+ * @param {Array} organFilters a list of organs for filtering the data
+ * @returns a list of QTL datasets
+ */
+export function getFilteredQtlData(data, organFilters) {
+  let filteredData = data;
+  if (organFilters.length > 0) {
+    filteredData = filteredData.filter((dataset) => {
+      const organsForDataset = dataset.biosample_ontology?.organ_slims
+        ? dataset.biosample_ontology.organ_slims
+        : [];
+      let hasOrgan = false;
+      for (let j = 0; j < organsForDataset.length; j++) {
+        if (organFilters.includes(organsForDataset[j])) {
+          hasOrgan = true;
+          break;
+        }
+      }
```

Comment:
`organsForDataset.some()` (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some) might make things more obvious about what you’re doing here, and `.some()` stops looping as soon as you return true once, so it’s as efficient as what you have.